examples/sum/script/src/main.rs

```diff
@@ -0,0 +1,45 @@
+use sphinx_sdk::{utils, ProverClient, SphinxProof, SphinxStdin};
+
+/// The ELF we want to execute inside the zkVM.
+const ELF: &[u8] = include_bytes!("../../program/elf/riscv32im-succinct-zkvm-elf");
+
+fn main() {
+    // Setup logging.
+    utils::setup_logger();
+
+    // Create an input stream and write input to it.
+    let nums = (0..1000).map(|i| i as u64).collect::<Vec<_>>();
```

Comment:
This is unidiomatic. I'd say `let nums: Vec<u64> = (0..1000).collect();`or `let nums = (0u64..1000).collect();`